tools/train.py

```diff
@@ -44,7 +44,7 @@ def parse_args():
         action='store_true',
         help='whether to set deterministic options for CUDNN backend.')
     parser.add_argument(
-        '--options', nargs='+', action=DictAction, help='arguments in dict')
+        '--update_config', nargs='+', action=DictAction, help='arguments in dict')
```

Comment:
Why don't you use `options` everywhere instead?

Reply:
`options` param is differently used in test.py and train.py. That's why I decided to introduce common parameter name for overriding configuration file fields. I can use whatever name but I would like to have the same meaning for train.py and test.py. What do you think?

Reply:
I see. Ok then.